tests/helpers/test_template.py

```diff
@@ -52,12 +51,12 @@ def _set_up_units(hass: HomeAssistant) -> None:
     """Set up the tests."""
     hass.config.units = UnitSystem(
         "custom",
-        accumulated_precipitation=LENGTH_MILLIMETERS,
+        accumulated_precipitation=UnitOfLength.MILLIMETERS,
```

Comment:
This is a unit of precipitation depth, not length.

Reply:
Good catch! I will make the requested change with a few UnitOfTime.Hour

Reply:
Requested change is implemented